plugins/module_utils/ome.py

```diff
@@ -426,6 +427,10 @@ def get_job_execution_details(self, job_id):
             pass
         return job_detail_status
 
+    def get_ome_version(self):
+        resp = self.invoke_request('GET', OME_INFO)
```

Comment:
Put this code in utils.py and not ome.py

Reply:
Done